tests/features/syntax_highlight_test.py

```diff
@@ -160,3 +161,12 @@ def test_syntax_highlighting_tabs_after_line_creation(run, tmpdir):
 def test_does_not_crash_with_no_color_support(run):
     with run(term='xterm-mono') as h, and_exit(h):
         pass
+
+
+@pytest.mark.parametrize('run_only_tmux', [run_tmux])
+def test_file_without_extension(run_only_tmux, tmpdir):
+    f = tmpdir.join('f')
+    f.write('#!/usr/bin/env python3')
+    f.chmod(777)
+    with run_only_tmux(str(f), term='screen-256color', width=25) as h, and_exit(h):
```

Comment:
`run_only_tmux` is almost always wrong, there's nothing that would require tmux in this test (but you shouldn't open the editor at all for this test anyway)